website/docs/components/accordion/partials/code/how-to-use.md

```diff
@@ -0,0 +1,125 @@
+## How to use this component
+
+The Accordion component is used to wrap and group together one or more AccordionItem child components. AccordionItems consist of “toggle” and “content” named blocks which can contain either plain text or HTML content.
+
+### Plain text content in toggle and content blocks
+```handlebars
+  <Hds::Accordion as |A|>
+    <A.Item>
+      <:toggle>Item one</:toggle>
+      <:content>
+        Additional content for item one
+      </:content>
+    </A.Item>
+    <A.Item>
+      <:toggle>Item two</:toggle>
+      <:content>
+        Additional content for item two
+      </:content>
+    </A.Item>
+  </Hds::Accordion>
+```
+
+### Complex HTML content in toggle and content blocks
+```handlebars
+  <Hds::Accordion as |A|>
+    <A.Item @isOpen={{true}}>
+      <:toggle>
+        <Hds::Alert @type="compact" @color="success" as |A|>
+          <A.Title>Title</A.Title>
+          <A.Description>Plan finished <small>22 days ago</small></A.Description>
+        </Hds::Alert>
+      </:toggle>
+      <:content>
+        <p class="hds-typography-body-200">
+          <strong>Queued:</strong>
+          9 days ago >
+          <strong>Finished:</strong>
+          9 days ago
+        </p>
+        <Hds::Table @caption="Example table">
+          <:head as |H|>
+            <H.Tr>
+              <H.Th>Name</H.Th>
+              <H.Th>Type</H.Th>
+              <H.Th>Value</H.Th>
+            </H.Tr>
+          </:head>
+          <:body as |B|>
+            <B.Tr>
+              <B.Td>Cell one A</B.Td>
+              <B.Td>Cell two A</B.Td>
+              <B.Td>Cell three A</B.Td>
+            </B.Tr>
+            <B.Tr>
+              <B.Td>Cell one B</B.Td>
+              <B.Td>Cell two B</B.Td>
+              <B.Td>Cell three B</B.Td>
+            </B.Tr>
+          </:body>
+        </Hds::Table>
+      </:content>
+    </A.Item>
+
+    <A.Item @containsInteractive={{false}} @isOpen={{true}}>
```

Comment:
[blocker] this actually contains an interactive element, which means it true. But in that case the example would be misleading (an accordion with mixed types of items? is it OK?)
Probably it would be better to split this example in two, with two types of toggles and contents.
```suggestion
    <A.Item @containsInteractive={{true}} @isOpen={{true}}>
```

Reply:
Thanks I missed that one when I was updating everything. I will change it locally as part of other changes.

Reply:
I think I will simplify the 2nd example